kvmagent/kvmagent/plugins/bmv2_gateway_agent/volume/helper.py

```
@@ -34,16 +34,27 @@ def _prepare_rbd_nbd():
+                    _rc, version = commands.getstatusoutput('rpm -qa %s' % fallback_pkg)
+                    if _rc or not version:
+                        raise exception.CephPackageNotFound(cmd=fallback_pkg)
+                    else:
+                        logger.info("Found %s package, version %s" % (fallback_pkg, version))
+                        actual_pkg = fallback_pkg
+                else:
+                    raise exception.CephPackageNotFound(cmd=pkg_name)
+            return version, actual_pkg
+
+        ceph_version, ceph_pkg = _get_package_version("ceph", "ceph-common", True)
+        current_rbd_nbd_version, rbd_nbd_pkg = _get_package_version("rbd-nbd")
```

Ruff (0.14.6)

Unpacked variable `rbd_nbd_pkg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)